### DISCUSSION TITLE
postcss7-compat and migrate to tailwindcss v3.0.15

### DISCUSSION THREAD

**Discussion by arishojaei** [tailwindlabs/tailwindcss#7180] [Help]
I used `npm:@tailwindcss/postcss7-compat@^2.0.2` in my **Vuejs** project and now I want to use TailwindCSS `v3.0.15` but I couldn't find any solution in TailwindCSS documentation!

Can anyone please show me a solution for that?

**Comment by adamwathan** [comment]
Tailwind 3 requires PostCSS 8+, so you will need to update the dependencies in your project. If you are using vue-cli you’ll have to look into the latest release of that project and see if they’ve updated to PostCSS 8 yet, they might still be on PostCSS 7 so if so you’ll have to stick with Tailwind 2 until that project is updated, or compile your Tailwind styles separately using Tailwind CLI.

**Comment by john-kurkowski** [comment]
[See this answer in a similar thread](https://github.com/tailwindlabs/tailwindcss/discussions/2854#discussioncomment-2047117). Vue 3, Tailwind 3, PostCSS 8, and Autoprefixer 10 all seem to work fine together. No more dependencies referencing branches, like tailwindcss/postcss7-compat.
